pom.xml

```diff
@@ -38,6 +38,7 @@
     <module>layrry-config-yaml</module>
     <module>layrry-config-toml</module>
     <module>layrry-launcher</module>
+    <module>layrry-cli</module>
```

Comment:
How about renaming "launcher" to "core" and "cli" to "launcher"? In the currently used terminology, "launcher" refers to the ready-made main class (as opposed to the API), so that'd be consistent.